docs/source/WritingTests.rst

```diff
@@ -957,6 +957,9 @@ Avocado offers some options for the test writers to skip a test:
 Test ``skip()`` Method
 ----------------------
 
+.. warning:: `self.skip()` is under deprecation process. Please adjust
+   your tests to use the `self.cancel()` or the skip decorators instead.
```

Comment:
Well at this point `self.cancel` is not supported during `setUp`, therefor people can't adjust their scripts. I know you are changing this later in this PR but I'd suggest changing the order of commits to first make `self.cancel` as powerful as `sefl.skip` and then add the deprecation message.